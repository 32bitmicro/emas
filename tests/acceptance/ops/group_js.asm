UJS 0
JLS 0
JES 0
JGS 0
JVS 0
JXS 0
JYS 0
JCS 0
