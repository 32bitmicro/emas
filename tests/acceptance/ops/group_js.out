@ 0x0000 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0001 : 0xe040  /  111 000 0 001 000 000  /  57408
@ 0x0002 : 0xe080  /  111 000 0 010 000 000  /  57472
@ 0x0003 : 0xe0c0  /  111 000 0 011 000 000  /  57536
@ 0x0004 : 0xe100  /  111 000 0 100 000 000  /  57600
@ 0x0005 : 0xe140  /  111 000 0 101 000 000  /  57664
@ 0x0006 : 0xe180  /  111 000 0 110 000 000  /  57728
@ 0x0007 : 0xe1c0  /  111 000 0 111 000 000  /  57792
